### PR TITLE
feat: gettext files (master, en, it)

### PR DESCRIPTION
**Pull request by didoda**
This provides translation files (created through `bin/cake gettext update`, updated manually with proper translations); and a basic handling of i18n inside bedita web (=> url to change session lang).

Still to better: managing plugins *.po files + AppController unit tests

### CODE REVIEW

File: src/Template/Element/Form/advanced_properties.twig [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
             {% set options = Schema.controlOptions(key, value, schema.properties[key]) %}
 
-            {{ Form.control(key, options)|raw }}
+            {{ Form.control(__(key), options)|raw }}
```
Comment (stefanorosanelli) [inline src/Template/Element/Form/advanced_properties.twig]: Not sure if we want to translate field names... some yes probably, but we have to think about it

File: src/Template/Element/Form/relations.twig [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
                     <header @click.prevent="toggleVisibility();" class="tab unselectable" v-bind:class="isLoading? 'is-loading-spinner' : ''">
                         <h2>
-                            {{ relationName|humanize|lower }}
+                            {{ __(relationName|humanize)|lower }}
```
Comment (stefanorosanelli) [inline src/Template/Element/Form/relations.twig]: I'm quite sure we don't want to translate relation names - we did not in BE3

File: src/Controller/ModulesController.php [hunk 3]
```diff
@@ -553,4 +554,19 @@ public function changeStatus() : ?Response
...
         return $this->redirect(['_name' => 'modules:list', 'object_type' => $this->objectType]);
     }
+
+    /**
+     * Change lang and redir to referer
+     *
+     * @param string $lang The lang code
+     * @return \Cake\Http\Response|null
+     */
+    public function lang($lang) : ?Response
```
[the remark this answers is not in the document]
Comment (batopa) [reply src/Controller/ModulesController.php]: Why don't we use cookie for store user lang configuring properly `I18Middleware`?

We could also use `I18n.switchLangUrl` to setup the switch url and avoid this method at all.

The configuration would be something as

```php
Configure::write('I18n', [
    'locales' => [
        'en_US' => 'en',
        'it_IT' => 'it',
    ],
    'default' => 'it',
    'languages' => [
        'en' => 'English',
        'it' => 'Italiano',
    ],
    'cookie' => [
        'name' => 'BEditaWebI18n',
        'create' => true,
    ],
    'switchLangUrl' => '/lang',
]);
```

In this case we should edit `composer.json` to require `"bedita/i18n": "^1.2"`

File: config/routes.php [hunk 4]
```diff
@@ -51,6 +52,9 @@
 
 Router::scope('/', function (RouteBuilder $routes) {
 
+    $locales = array_keys(Configure::read('I18n.languages'));
+    $lang = implode('|', $locales);
```
[the remark this answers is not in the document]
Comment (didoda) [reply config/routes.php]: `$locales` and `$lang` can be removed

File: src/Template/Element/Form/advanced_properties.twig (hunk 1 above)
Comment (didoda) [reply src/Template/Element/Form/advanced_properties.twig]: Fair enough. I'd say that we should at least translate `title` and `description`.

File: src/Controller/ModulesController.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (didoda) [reply src/Controller/ModulesController.php]: Good point. I'll try to use cookie

File: src/Template/Element/Menu/project.twig [hunk 5]
```diff
@@ -5,8 +5,15 @@
     {#{% element 'tree' %}#}
     {% element 'Menu/colophon' %}
 
+
     {% if user %}
     <ul class="menu-links">
+        {# Localization link(s) #}
+        {% for code, name in I18n.getLanguages() if code != I18n.getLang() %}
+            <li>
+                {{ Html.link(name, I18n.changeUrlLang(code, config('I18n.switchLangUrl')))|raw }}
```
Comment (batopa) [inline src/Template/Element/Menu/project.twig]: Add `, { 'title': name }` as in `menu.twig`

Comment (batopa) [reply src/Template/Element/Menu/project.twig]: Nevermind, my mistake

**Pull request merged**